xc7/tests/ff_sr_ce/generate.py

```diff
@@ -1,13 +1,29 @@
+""" Script specialize verilog localparam's and $dumpfile directives.
```

Comment:
I don't quite understand why you can't just pass the two values in as either Verilog defines or other arguments?

Reply:
It it just easier form a cmake perspective or something?

Reply:
Because that is *way* harder.  For example, the testbench output wave file needs to be changed, and Verilog 2005 doesn't have proper STRINGIFY support.

Reply:
> It it just easier form a cmake perspective or something?

In some ways.  Consider that these source files have to go to three different tools, yosys, iverilog and SymbiYosys.

Reply:
Okay, SGTM.